sklearnex/linear_model/tests/test_linear.py

```diff
@@ -81,3 +82,46 @@ def test_sklearnex_import_elastic():
     assert "daal4py" in elasticnet.__module__
     assert_allclose(elasticnet.intercept_, 1.451, atol=1e-3)
     assert_allclose(elasticnet.coef_, [18.838, 64.559], atol=1e-3)
+

```

Comment:
I would actually recommend to also check for such small numbers in an actual algorithm implementation in order to not get errors while inverting values